txtl-liposome.md

```diff
@@ -0,0 +1,69 @@
+# TX-TL Liposome Protocol
+
+This protocol describes how to create liposomes that contain TX-TL
+inside of a lipd-based container.  The liposomes created by this
+protocols are between 1 and 10 um in diameter and should be usable
+with an cell-free protein expression system.
+
+## Step 1: Liposome-oil emulsion
+
+1. Mix mineral oil by gentle inversion before use (Mineral oil can
+   dissolve lipid better than paraffin)
+2. Place 0.5 mL of mineral oil in the vials
+3. Vortex for 10 mins
+4. Incubate vials at 50C for 10mins
+5. Repeat 50C incubation and vortexing until the lipid is completely
+   dissolved.
+
+### Notes:
+
+* After 3 rounds of this I put the vials to incubate at 60C for 3.5
+  hours then vortexed it, the mixture looks a little different than
+  before. The consistency looked soft.
+
+* 28 March 2018: What helps in later trials was leaving it in heated @
+  60C sonicator bath for 1 hr.
+
+## Step 2: Making liposomes
+
+6. Place 20 μL of centrifuge buffer (PBS + 300 mM sucrose, pH 8) into
+   a labeled eppendorf tube
+
+7. Add 2 μL of reaction into 300 μL phospholipid solution, vortex for
+   30 s to create an emulsion
+
+8. Add the emulsion on top of the 20 μL centrifuge buffer, wait for the interface to stabilize and flatten between the emulsion and buffer
+
+9. Centrifuge at 11000 RPM for 30 s (tabletop centrifuge works well)
+
+10. Carefully remove the mineral oil, take 10 μL of the centrifuge
+    buffer (where the vesicles have been formed)
+
+11. Add 10 μL of solution (where vesicles hopefully \"live\") to 10 μL
+    of final buffer (50 mM HEPES + 300 mM sucrose, pH 8), careful not
+    to touch the sides of tube
+
+12. Mix gently, pipetting up and down with a new tip"
+
+## Step 3: Microscopy 
+13. Use Frameseal to make a small chamber on a coverslip
+
+14. Add 10 μL of the final solution to the inside chamber and seal
+    with cover slip
+
+15. Observe on microscope
+
```

Comment:
![20180208_201549](https://user-images.githubusercontent.com/32440023/43659506-1a009982-9711-11e8-8548-e11bbb5b7b1b.jpg)
 Image on 60X  RFP channel  (left) and bright field (right)